testing/trino-product-tests/src/main/java/io/trino/tests/product/jdbc/TestExternalAuthorizerOAuth2.java

```diff
@@ -126,6 +126,7 @@ private WebDriver getWebDriver()
     {
         ChromeOptions options = new ChromeOptions();
         options.setAcceptInsecureCerts(true);
+        options.addArguments("--disable-gpu");
```

Comment:
Should there be a TODO to remove this once resolved upstream? Or does it not matter?

Reply:
A TODO might be a good option here, but it also may not matter. Given that it's docker I can't imagine it'll ever actually use the GPU. I could go either way; I'll add the TODO tomorrow if this PR hasn't been merged as I'm dog tired and done for today.

Reply:
Sure, no problem. Also, can you please add the reason in the commit message too.

Reply:
So Selenium just released a new version of the Chrome container so I've just removed the work around and only reenabled the product tests.